tests/config/test_themes.py

```diff
@@ -49,27 +51,53 @@ def test_complete_and_incomplete_themes():
     assert result == complete_and_incomplete_themes()
 
 
+class theme:
+    class Color(Enum):
+        WHITE__BOLD = "white          #fff   #ffffff , bold"
+        WHITE__BOLD_ITALICS = "white  #fff   #ffffff , bold , italics"
+        DARK_MAGENTA = "dark_magenta  h90    #870087"
+
+    STYLES = {
+        "s1": (Color.WHITE__BOLD, Color.DARK_MAGENTA),
+        "s2": (Color.WHITE__BOLD_ITALICS, Color.DARK_MAGENTA),
+    }
```

Comment:
Is there a reason the elements of this shouldn't be in the test below?

Note that with the style decoupling you won't need the wrapping class.

Reply:
@neiljp Not sure I get what you mean here about elements, I've decoupled the function though.